### DISCUSSION TITLE
Switch window visibility on keybind press

### DISCUSSION THREAD

**Discussion by Bamboooz** [emilk/egui#4797] [Q&A] [answered]
Hi. Is there a way that I could make my app window toggle its visibility on pressing Alt+T? I'm trying to make something simillar to the NVIDIA performance overlay and i need a way to toggle it on/off. Im using standard eframe.

![obraz](https://github.com/emilk/egui/assets/83116089/66b8b85f-bc69-49d6-8fbc-e5454bb4c9b2)

**Comment by emilk** [comment, marked as answer]
You can send a https://docs.rs/egui/latest/egui/viewport/enum.ViewportCommand.html#variant.Visible to change visibility

**Comment by Bamboooz** [comment]
Yep, just did

```rs
ctx.send_viewport_cmd(egui::viewport::ViewportCommand::Visible(false));
```

worked as I wanted, thanks!
